scss/components/_soe_news.scss

```diff
@@ -99,11 +99,15 @@
   }
 }
 
+div#block-ds-extras-banner {
```

Comment:
It is discouraged to combine element and id/class selectors. The `#` selector should be plenty strong enough.